.github/workflows/cloud-workflow.yml

```diff
@@ -172,6 +172,36 @@ jobs:
           SLACK_ICON_EMOJI: ":boom:"
           SLACK_COLOR: "#FF0000"
           SLACK_FOOTER: ' '
+  cloud-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Publish SwaggerHub API
+        env:
+          MAGMA_ROOT: "${{ github.workspace }}"
+          SWAGGERHUB_API_KEY: "${{ secrets.SWAGGERHUB_API_KEY }}"
+        run: |
+          cd ${MAGMA_ROOT}
```

Comment:
Do you really need to be in MAGMA_ROOT to execute those commands?

Reply:
Not really, thanks for the catch!